test/test_client.py

```diff
@@ -474,6 +481,81 @@ def test_set_plc_datetime(self):
         # Can't actual set datetime in emulated PLC, get_plc_datetime always returns system time.
         # self.assertEqual(new_dt, self.client.get_plc_datetime())
 
+    def test_wait_as_completion_pass(self, timeout=10):
+        # Cli_WaitAsCompletion
+        # prepare Server with values
+        area = snap7.types.areas.DB
+        dbnumber = 1
+        size = 1
+        start = 1
+        data = bytearray(size)
+        self.client.write_area(area, dbnumber, start, data)
+        # start as_request and test
+        p_data = self.client.as_db_read(dbnumber, start, size)
+        try:
+            self.client.wait_as_completion(ctypes.c_ulong(timeout))
```

Comment:
this is not required anymore then right

Reply:
`wait_as_completion()` works a bit different than `as_check_complete()`. It realy has to throw an exception in case stuff has timeouted, which is an Snap7Excpetion.
But I see, that other Snap7Exceptions are being ignored, will fix that.

Reply:
... but this test should be outdated... I splitted it into two tests.